Blammo/src/Blammo/Logging/ThreadContext.hs

```diff
@@ -0,0 +1,10 @@
+module Blammo.Logging.ThreadContext
+  ( MonadMask
+  , withThreadContext
+  , myThreadContext
+  , Pair
```

Comment:
It's a bit irksome that the "pairs" in `Message` are `SeriesElem`, but the "pairs" in thread context are `Pair`.